account_cutoff_prepaid/account_cutoff_view.xml

```diff
@@ -54,6 +55,7 @@
     <field name="arch" type="xml">
         <field name="parent_id" position="after">
             <field name="move_line_id" invisible="'prepaid' not in context.get('type', '-')"/>
+            <field name="move_line_period_id" invisible="'prepaid' not in context.get('type', '-')"/>
```

Comment:
@Olivier-LAURENT so far, no performance issue even on very large databases, and no user requirement for sorting (yet).